jvm/sargon-android/build.gradle.kts

```diff
@@ -157,6 +157,12 @@ android.libraryVariants.all {
         "generate${buildTypeUpper}UniFFIBindings",
         Exec::class
     ) {
+        val rebuild = properties["rebuild"] ?: true
```

Comment:
If we go with this way, I would rename it to `regenerate` and not `rebuild` since this task just generates code and does not build anything.

But regardless this is a param for us to use when we know that the build didn't produce any changes. As a gradle user this is not the correct way to do it. 

Do we really need this for now? If so yes, we can "rely" on it for now but such mechanism is not reliable. We can make gradle do this automatically with `upToDateWhen`.

Reply:
I renamed the param to `regenerate`. We should spend more time on this to have a proper solution in place. This quick solution on the other hand will help us shorten the time spent on consecutive builds.